tests/integration_tests/build/test_coverage.py

```diff
@@ -25,15 +25,15 @@ def is_on_skylake():
 # differences may appear.
 if utils.is_io_uring_supported():
     COVERAGE_DICT = {
-        "Intel": 81.71 if is_on_skylake() else 82.23,
-        "AMD": 80.81,
-        "ARM": 82.69,
+        "Intel": 81.49 if is_on_skylake() else 82.00,
```

Comment:
I suspect that the benchmark code itself is categorized as prod code, not test code. So as the benchmark code grows, so does the code volume target. I have not confirmed this though, but given where this PR introduces code changes, I think it makes sense. The other possibility as to the amount of coverage dropped is in the duplication of the test data constants/builders. I am hoping this gap will drop a bit when I fix in the next push.

Reply:
Let's see what it looks like after removing the duplicate code. I doesn't look natural to me that benchmark code pulls our coverage down, to be honest. If it's the case, we should do something to fix it.

Reply:
I agree it does not make sense. My current theory is that any code not under `#[test] ` counts as code that needs coverage. Configurability to fix this would improve our coverage accuracy/relevancy.

Reply:
There is an improvement with the removed duplication [CI build after cleanup](https://buildkite.com/firecracker/firecracker-pr/builds/2355). There is still a slight drop though as I expected.

Reply:
Yeah, the drop looks negligible now.